deployments/helm/vpp-icmp-responder/templates/vppagent-icmp-responder-nse.tpl

```diff
@@ -29,9 +29,11 @@ spec:
               topologyKey: "kubernetes.io/hostname"
       containers:
         - name: icmp-responder-nse
-          image: {{ .Values.registry }}/networkservicemesh/vppagent-icmp-responder-nse:{{ .Values.tag }}
+          image: {{ .Values.registry }}/networkservicemesh/vpp-test-common:{{ .Values.tag }}
           imagePullPolicy: {{ .Values.pullPolicy }}
           env:
+            - name: VPP_APP
+              value: "vppagent-icmp-responder-nse"
```

Comment:
Do we need a command here to choose NSE application to run?

Reply:
No, all VPP-based images use predefined entrypoint. Application is selected using the `VPP_APP` environment variable.